js/main.js

```diff
@@ -4,7 +4,8 @@ import './form-validation.js';
 import './scale-control.js';
 import './effects-control.js';
 import {getData} from './api.js';
-import {showAlert} from './utils.js';
+import {showAlert, debounce} from './utils.js';
+import {init, getSortingPhotos} from './sorting.js';
 
 getData()
 	.then(renderedPhotos)
```

Comment:
У тебя здесь уже есть вызов `getData()` при том лучше так, потому что top level await появился недавно, поэтому всю логику что ниже можно перенести сюда:
```suggestion
	.then((photos) => {
		const debouncedRenderGallery = debounce(renderedPhotos);
		init(photos, debouncedRenderGallery);
		renderedPhotos(photos);
	})
```